app/frontend/actions.js

```diff
@@ -122,10 +169,26 @@ module.exports = ({setState, getState}) => {
       const walletIsLoaded = true
       const ownAddressesWithMeta = await wallet.getFilteredVisibleAddressesWithMeta()
       const transactionHistory = await wallet.getHistory()
+      const delegationHistory = await wallet.getDelegationHistory()
       const balance = await wallet.getBalance()
+      const rewards = await wallet.getRewards()
       const conversionRates = getConversionRates(state)
       const sendAmount = {fieldValue: '', coins: 0}
       const sendAddress = {fieldValue: ''}
+      const stakePoolID = 0
+      const stakePools = [
+        {
+          address: ADALITE_CONFIG.ADALITE_STAKE_POOL_ADDRESS,
```

Comment:
what is STAKE_POOL_ADDRESS? Stake pools *do not* have addresses, only IDs

Reply:
Renamed in all occurrences.